roles/openshift_master/tasks/main.yml

```diff
@@ -131,6 +131,9 @@
 - name: Install the systemd units
   include: systemd_units.yml
 
+- name: Install Master system container
+  include: system_container.yml
```

Comment:
would suggest putting the common when clauses in system_container.yml on this include.